### ISSUE TITLE
SurveyForDesigner instances don't have renderedElement property anymore

### ISSUE THREAD

**Issue by xadie** [surveyjs/survey-creator#951]
### Are you requesting a feature, reporting a bug or ask a question?
req. feature / rep. bug?

### What is the current behavior?
from version 1.7.24 on SurveyForDesigner instances don't expose renderedElement property anymore.

### What is the expected behavior?
that they have like page, panels and question the renderedElement property that points to the rendered html element container.

The main motivation was that i used it as easy access to query for the corresponding title-adorner html element, to apply style changes depending on entered title-values.

### How would you reproduce the current behavior (if this is a bug)?
listening on the event like onModified / onCustomError and checking on the options.target / options.obj argument.

#### Provide the test code and the tested page URL (if applicable)
https://plnkr.co/edit/Powa86ZDcxaC0sdR

**Comment by dmitrykurmanov** [comment]
I am not sure about supporting `renderedElement` in v1.7.23. Please check the example with 1.7.23 https://plnkr.co/edit/0kunZjNQ7DP1iu5m there is no `renderedElement` property either.

**Comment by xadie** [comment]
actually you're plunker example is showing that renderedElement is existing on v1.7.23. ;)
your `console.dir('Where did it went?', options.target.renderedElement);` is evaluated to 
`Where did it went? <div class="svd_surveyjs_designer_container">` on the console.

**Comment by andrewtelnov** [comment]
@xadie I remember we have changed something here some times ago. We started use knockout components more extensively. It was an internal property. What is your goal? 

Thank you,
Andrew

**Comment by xadie** [comment]
i used it mainly as a quick access to the title adorner html element of each question element (including pages and survey) to change the style depending on value changes to its title property.

**Comment by tsv2013** [comment]
You need to use the https://surveyjs.io/Documentation/Survey-Creator?id=surveycreator#onAdornerRendered event to access adorner markup

**Comment by xadie** [comment]
yeah that is the workaround for me - but that means I have to manage the references of the adorner for the element that needs to be updated as the event `onAdornerRendered` is only fired on render. Going over renderedElement  property is way more convenient, as that is accessible over the options properties of events like `onModified` or `onPropertyValidationCustomError` and so on.
